src/QueryEditor/EventAssetProperties.tsx

```diff
@@ -106,6 +107,7 @@ export const EventAssetProperties = (props: Props): React.JSX.Element => {
         hideGroupBy={props.queryType === 'simple'}
         hideTagFilter={props.queryType === 'simple'}
         hideAdvancedOptions={props.queryType === 'simple'}
+        hideValueFilter={true}
```

Comment:
why hide them here?

Reply:
it could be usefull to have value filters here

Reply:
while you are at it, also enable the tag filter. But make use by default status == Good is here